releasenotes/notes/synth_permutation_depth_lnn_kms-c444f3a363f3a903.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade_synthesis:
+  - |
+    Port :func:`.synth_permutation_depth_lnn_kms`, used to synthesize qubit permutations, to Rust.
```

Comment:
maybe expand this a bit to: "used to synthesize permutations _for linear connectivity_, to Rust."